shamir_mnemonic/__init__.py

```diff
@@ -638,16 +658,18 @@ def generate_mnemonics_random(
     )
 
 
-def combine_mnemonics(mnemonics, passphrase=b""):
+def combine_mnemonics(mnemonics):
     """
```

Comment:
This changes the high level API which, as I explained elsewhere, is something I would like to avoid if we can.

Reply:
well, yes, that's what we are doing, per https://github.com/trezor/trezor-firmware/issues/407
If you have specific remarks, please elaborate in this issue or in the other one.